config/app.php

```diff
@@ -132,7 +134,7 @@
     |   - Amazon ELB users should always use the "HEADER_X_FORWARDED_AWS_ELB" option.
     */
 
-    'trustedProxyHeaders' => Illuminate\Http\Request::HEADER_X_FORWARDED_ALL,
+    'trustedProxyHeaders' => 'HEADER_X_FORWARDED_ALL',
```

Comment:
The constant has been completely removed.

Reply:
Looks like the recommendation is to use the full definition of constants instead: https://github.com/symfony/symfony/blob/4537f85963fba75f9ec2aa9a39c048140260f1f9/src/Symfony/Component/HttpFoundation/Request.php#L50-L51

So I think the value would need to be `'trustedProxyHeaders' => Illuminate\Http\Request::HEADER_X_FORWARDED_FOR | Illuminate\Http\Request::HEADER_X_FORWARDED_HOST | Illuminate\Http\Request::HEADER_X_FORWARDED_PORT | Illuminate\Http\Request::HEADER_X_FORWARDED_PROTO`

Reply:
That's what it's mapped to in the middleware in the storm repo, I'm switching to a default of the string constant instead because we've already been burnt by them removing a constant which we have no way to polyfill back in, so all default configs from 1.1.4 to 1.1.8 will be forced to make a configuration file change when upgrading to 1.2 because of it or else their entire site will break because the Symfony / Laravel controlled constant no longer exists.